cmd/flags.go

```diff
@@ -249,6 +250,12 @@ var (
 		Usage: "Specifies the size in bytes of bolt db's mmap syscall allocation",
 		Value: 536870912, // 512 Mb as a default value.
 	}
+	// ApiTimeoutFlag specifies the timeout value for API requests in minutes. A timeout of zero means no timeout.
+	ApiTimeoutFlag = &cli.IntFlag{
+		Name:  "api-timeout",
+		Usage: "Specifies the timeout value for API requests in minutes",
```

Comment:
do users want minutes or seconds here? I can imagine a situation where users may want to use this flag to restrict calls to be less than a minute or fail